core/embed/sec/tropic/inc/sec/tropic_transport.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * This file is part of the Trezor project, https://trezor.io/
+ *
+ * Copyright (c) SatoshiLabs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef TREZORHAL_TROPIC_TRANSPORT_H
+#define TREZORHAL_TROPIC_TRANSPORT_H
+
+#include <trezor_types.h>
+#include "libtropic.h"
```

Comment:
I think this include is not necessary here and can be moved to `tropic_transport.c`.

Reply:
Yep https://github.com/trezor/trezor-firmware/pull/4172/commits/07425d69411a20883084721064158b4c5a70e8e5#diff-55fbcf8fbcee5ed9d0ad61ec900e32043ee8b7f6e77f75d9921410bb55d63decL24